scripts/dash-docking-exploration.py

```diff
@@ -0,0 +1,227 @@
+        html.Div(
+            [
+                html.H4("By Compound"),
+                dcc.Dropdown(
+                    by_compound.columns,
+                    "POSIT_R_Good",
+                    id="cmpd-bar-xaxis",
+                ),
+                dcc.Graph(id="per-compound-bar-chart"),
+            ],
+            style={"width": "49%", "display": "inline-block", "float": "right"},
+        ),
+        html.Div(
+            [
+                html.Div(
+                    [
+                        html.H4("X-Axis"),
+                        dcc.Dropdown(
+                            by_structure_tidy["variable"].unique(),
+                            "Resolution",
+                            id="crossfilter-xaxis-column",
+                        ),
+                        dcc.RadioItems(
+                            ["Linear", "Log"],
+                            "Linear",
+                            id="crossfilter-xaxis-type",
+                            labelStyle={
+                                "display": "inline-block",
+                                "marginTop": "5px",
+                            },
+                        ),
+                    ],
+                    style={
+                        "width": "49%",
+                        "float": "top",
+                        "display": "inline-block",
+                    },
+                ),
+                html.Div(
+                    [
+                        html.H4("Y-Axis"),
+                        dcc.Dropdown(
+                            by_structure_tidy["variable"].unique(),
+                            "POSIT_R_Good",
+                            id="crossfilter-yaxis-column",
+                        ),
+                        dcc.RadioItems(
+                            ["Linear", "Log"],
+                            "Linear",
+                            id="crossfilter-yaxis-type",
+                            labelStyle={
+                                "display": "inline-block",
+                                "marginTop": "5px",
+                            },
+                        ),
+                    ],
+                    style={
+                        "width": "49%",
+                        "float": "bottom",
+                        "display": "inline-block",
+                    },
+                ),
+            ],
+            style={
+                "padding": "10px 5px",  # "float": "left",
+                # "display": "inline-block",
+            },
+        ),
+        html.Div(
+            [
+                dcc.Graph(
+                    id="crossfilter-indicator-scatter",
+                    # hoverData={"points": [{"customdata": "Japan"}]},
+                )
+            ],
+            style={
+                # "width": "49%",
+                "display": "inline-block",
+                # "float": "right",
+                "padding": "0 20",
+            },
+        ),
+    ]
+)
```

Comment:
How is this different from the similar code in the other script?

We probably want to have this in a separate file that we can just import, such that this script is easier to read and handle, when needed. And reuse it that way, if possible.